apollo-client/src/main/java/com/ctrip/framework/apollo/util/ExceptionUtil.java

```diff
@@ -52,11 +52,19 @@ public static String getDetailMessage(Throwable ex) {
         counter--;
         continue;
       }
-      builder.append(" [Cause: ").append(cause.getMessage());
+      builder.append(" [Cause: ")
+              .append(getThrowingClassName(cause))
```

Comment:
It looks like not necessary to extract a method for this?
```suggestion
              .append(cause.getClass().getSimpleName())
```